asreviewcontrib/insights/utils.py

```diff
@@ -1,8 +1,33 @@
 import numpy as np
+from asreview import open_state
+from asreview.state import SQLiteState
 
 
 def get_labels(state_obj, priors=False):
```

Comment:
Maybe better to keep this function private. To avoid users using this module for this function in particular.